OasisDEX/oasis-borrow
# [L2] Add "Network" row to the product card

# [[L2] Add "Network" row to the product card](https://app.shortcut.com/oazo-apps/story/8731/l2-add-network-row-to-the-product-card)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
- some minor fixes with the network names, so we wont use `networksByName['ethereumMainnet']` cause its ugly
- added network row to the product cards (feature togglified)
  
## How to test 🧪
- enable `UseNetworkRowProductCard`
- check out the product cards
